lib/react_client/react_interop.dart

```diff
@@ -460,20 +422,20 @@ abstract class ReactDartComponentVersion {
 @anonymous
 class ReactClassConfig {
   external factory ReactClassConfig({
-    String displayName,
-    List mixins,
-    Function componentWillMount,
-    Function componentDidMount,
-    Function componentWillReceiveProps,
-    Function shouldComponentUpdate,
-    Function componentWillUpdate,
-    Function componentDidUpdate,
-    Function componentWillUnmount,
-    Function getChildContext,
-    Map<String, dynamic> childContextTypes,
-    Function getDefaultProps,
-    Function getInitialState,
-    Function render,
+    String? displayName,
+    List? mixins,
+    Function? componentWillMount,
+    Function? componentDidMount,
+    Function? componentWillReceiveProps,
+    Function? shouldComponentUpdate,
+    Function? componentWillUpdate,
+    Function? componentDidUpdate,
+    Function? componentWillUnmount,
+    Function? getChildContext,
+    Map<String, dynamic>? childContextTypes,
+    Function? getDefaultProps,
+    Function? getInitialState,
+    Function? render,
```

Comment:
Is it worth adding function arity to the function types here?

Reply:
This API is deprecated only has a couple remaining usages, so I don't think it's worth it.